packages/shared/src/graphql/fragments.ts

```diff
@@ -143,7 +143,6 @@ export const SOURCE_BASE_FRAGMENT = gql`
     }
     memberPostingRole
     memberInviteRole
-    moderationRequired
```

Comment:
This is intented?

Reply:
Not really, this PR was from before this field was added, adding it again. Thanks for spot it!